.docker/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
There's now no point in this file right?

Reply:
It was to have something that didn't call `uvicorn` with the `--reload` parameter, as well as specify the host to `0.0.0.0`